api.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+var conf = require('./server/config')
+var apm = require('elastic-apm-node').start(conf.apm)
```

Comment:
@watson This currently shares the apm configs with the parent app. Should I give it a different `serviceName` though?

Reply:
Yes, we want it to have a different service name to show up separately in the service map UI